src/components/DataSubmissions/UserGuide.tsx

```diff
@@ -23,7 +23,7 @@ export const UserGuide: FC = () => (
   <StyledText variant="body1">
     Prior to beginning uploading process, read detailed instructions available in the{" "}
     <Link to={Guide} target="_blank">
```

Comment:
The User Story used to specify that we also update this link to the same one in the Navbar. Could you just double check with Kailing if this requirement was intentionally removed? The PDF here is essentially a placeholder.